fe/fe-core/src/main/java/org/apache/doris/catalog/Table.java

```diff
@@ -511,4 +511,22 @@ public Set<String> getPartitionNames() {
     public AnalysisJob createAnalysisJob(AnalysisJobScheduler scheduler, AnalysisJobInfo info) {
         throw new NotImplementedException();
     }
+
+    /**
+     * for NOT-ANALYZED Olap table, return estimated row count,
+     * for other table, return 1
```

Comment:
should we return nagtive number and generate a default row number when derive stats?